Package.resolved

```diff
@@ -0,0 +1,86 @@
+{
+  "pins" : [
```

Comment:
When I add SwiftLint and it's build tool as dependencies locally I don't get these additional dependencies. Where are these coming from and what requires them?

Reply:
Mmm I'm checking and in the [Swiftlint](https://github.com/realm/SwiftLint) repository the [Package.resolved](https://github.com/realm/SwiftLint/blob/main/Package.resolved) file also contains these libraries. Why it doesn't show for you locally in your project... I don't know 😅
Maybe "sub-dependencies" aren't shown"?